### DISCUSSION TITLE
Resolved

### DISCUSSION THREAD

**Discussion by oliviaz134** [community/community#139249] [GitHub Education]

**Comment by alx-mp4** [comment]
Hi @oliviaz134, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.
